backend/app/api/endpoints/installed_plugins.py

```
@@ -0,0 +1,134 @@
+        db=db,
+        user_id=current_user.id,
+    )
+
+
+@router.post(
+    "/upload",
+    response_model=InstalledPlugin,
+    status_code=status.HTTP_201_CREATED,
+)
+async def upload_plugin(
+    file: UploadFile = File(...),
```

Ruff (0.15.15)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)